tests/cli/test_cmds_validate.py

```diff
@@ -192,6 +192,52 @@ def test_validate_min_max_instances_success(
     )
 
 
+@patch("paasta_tools.cli.cmds.validate.get_file_contents", autospec=True)
+def test_validate_schema_min_max_instances_absent(mock_get_file_contents, capsys):
+    kubernetes_content = """
+invalid-max-only:
+  cpus: 0.1
+  max_instances: 3
+  mem: 250
+  disk: 512
+  cmd: virtualenv_run/bin/python adindexer/adindex_worker.py
+"""
+    mock_get_file_contents.return_value = kubernetes_content
```

Comment:
we could probably store this as a dict and use `yaml.dump(kubernetes_content)` as the value of `mock_get_file_contents.return_value`, but that's pretty nit-picky so feel free to ignore this unless you like the dict form more :p

e.g,:
```suggestion
    kubernetes_content = {
        "invalid-max-only": {
            "cpus": 0.1
            "max_instances": 3
            "mem": 250
            "disk": 512
            "cmd": "virtualenv_run/bin/python adindexer/adindex_worker.py"
        }
    }
    mock_get_file_contents.return_value = yaml.dump(kubernetes_content)
```

Reply:
(it would also be nice to make the mocking here use context managers as we usually do in newer tests/internally - but that's probably fine to ignore as well since every other test in this file is using the decorator approach to mocking)